.github/workflows/release.yml

```diff
@@ -30,11 +28,6 @@ jobs:
         uses: actions/checkout@v2
         with:
           path: repo
-      - name: Install Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: ${{ env.rustToolchain }}
-          override: true
```

Comment:
Rust is needed for the release workflow too, as it builds the engine distribution. Please do not remove it.